src/VirtoCommerce.CatalogModule.Data/Repositories/CatalogRepositoryImpl.cs

```diff
@@ -440,6 +440,13 @@ public async Task RemoveAllPropertyValuesAsync(string propertyId)
             }
         }
 
+        public async Task RemoveAssociationsAsync(string[] ids)
+        {
+            var itemsForRemove = DbContext.Set<AssociationEntity>().Where(x => ids.Contains(x.Id));
```

Comment:
I think we do not need a specific Remove method for associations (specific are with SQL), it could be implemented directly in the service, like here: https://github.com/VirtoCommerce/vc-module-catalog/blob/89c0ce5403eaff38cabb1b9976c9d8c5898b8b33/src/VirtoCommerce.CatalogModule.Data/Services/PropertyDictionaryItemService.cs#L81.